tests/test_async_patterns.py

```
@@ -143,44 +159,61 @@ async def test_timeout():
+
+                        # Mock certifi.where()
+                        with patch("certifi.where", return_value="/fake/cert/path"):
+                            # Mock a slow Matrix operation
+                            with patch(
+                                "mmrelay.matrix_utils.AsyncClient"
+                            ) as mock_client_class:
+                                mock_client = MagicMock()
+                                mock_client.rooms = {}
+
+                                # Make sync take longer than timeout
+                                async def slow_sync(*args, **kwargs):
```

Ruff (0.14.14)

[warning] Unused function argument: `args`

(ARG001)